app/uk/gov/hmrc/apiplatform/modules/mfa/controllers/profile/MfaController.scala

```diff
@@ -120,22 +139,61 @@ class MfaController @Inject()(
   }
 
   def nameChangeAction(mfaId: MfaId): Action[AnyContent] = atLeastPartLoggedInEnablingMfaAction { implicit request =>
-
+    val errorMessage = "Failed to change MFA name"
     MfaNameChangeForm.form.bindFromRequest.fold(
       form => Future.successful(BadRequest(nameChangeView(form, mfaId))),
       form => {
         thirdPartyDeveloperMfaConnector.changeName(request.userId, mfaId, form.name) map {
           case true => Redirect(uk.gov.hmrc.apiplatform.modules.mfa.controllers.profile.routes.MfaController.authAppSetupCompletedPage())
-          case false => InternalServerError("Failed to change MFA name")
+          case false => InternalServerError(errorHandler.standardErrorTemplate(errorMessage, errorMessage, errorMessage))
```

Comment:
This pattern is used several times. Can the duplication move to a common method?

Reply:
Addressed

Reply:
Addressed